ls1intum/Artemis
# Instructor Course Dashboard: Show Submissions Button for Programming Exercises

### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) ~~on the test server https://artemistest.ase.in.tum.de~~ locally.

### Motivation and Context
The Assessment Dashboard shows the Submissions-Button for every exercise type (including programming exercises), the Instructor Course Dashboard did not. This PR makes this consistent and always shows the Submissions-Button.

### Steps for Testing

0. You need a course with exercises of every type and a user with instructor rights
1. Go to the Instructor Course Dashboard
2. Check that for every exercise the Submissions-Button is shown
3. Check that clicking the button opens the correct page


### Screenshots
old:
![image](https://user-images.githubusercontent.com/26540346/117267856-fe315b00-ae56-11eb-92e6-623a34fa8f04.png)

new:
![image](https://user-images.githubusercontent.com/26540346/117267911-0d180d80-ae57-11eb-837b-3629ce4f5c5a.png)
